invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RemoveFromCommunity/RemoveFromCommunityAction.js

```diff
@@ -136,7 +137,7 @@ export class RemoveFromCommunityAction extends Component {
             {error && (
               <ErrorMessage
                 header={i18next.t("Something went wrong")}
-                content={error.message}
+                content={error.response?.data?.errors[0]?.message || error.message}
```

Comment:
I tested errors property, but  I got an error, as it looks in the FieldErrorList component for `messages` rather than `message`, so I assumed that this is been designed for another type of errors (form validation errs), see https://github.com/inveniosoftware/react-invenio-forms/blob/master/src/lib/elements/ErrorMessage.js#L20
Alternatively, we could submit a pull request to address this issue on react-invenio-forms in that case?

Reply:
I created a PR for that here: https://github.com/inveniosoftware/react-invenio-forms/pull/202